x/concentrated-liquidity/architecture.md

```diff
@@ -292,15 +276,17 @@ This message should call the `createPosition` keeper method that is introduced i
 This message allows LPs to withdraw their position in a given pool and range (given by ticks), potentially in partial
 amount of liquidity. It should fail if there is no position in the given tick ranges, if tick ranges are invalid,
 or if attempting to withdraw an amount higher than originally provided. If an LP withdraws all of their liquidity
-from a position, the collectFees method is called and then the position is deleted from state.
+from a position, then the position is deleted from state. However, the fee accumulators associated with the position
+are still retained until a user claims them manually.
```

Comment:
It's still the case. I think we might want to keep this as is because what if I don't want to remove liquidity but not claim rewards at the same time? For example, for tax reasons.

I think having LPing and fee collection logic might actually be beneficial. One can always just issue another message for collecting fees if needed.

Reply:
I'm going to merge this. Happy to continue discussing separately